types/prismic-dom/index.d.ts

```diff
@@ -1,12 +1,50 @@
-// Type definitions for prismic-dom 2.0
+// Type definitions for prismic-dom 2.1
 // Project: https://github.com/prismicio/prismic-dom#readme
 // Definitions by: Nick Whyte <https://github.com/nickw444>
 //                 Siggy Bilstein <https://github.com/sbilstein>
+//                 Douglas Nomizo <https://github.com/douglasnomizo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+type ElementType =
```

Comment:
It looks like `prismic-rich-text`has some type definitions already. Did you consider including those somehow? 
https://github.com/prismicio/prismic-richtext/blob/master/d.ts/elements.d.ts

Reply:
It's actually a simplified version of the definition in the `prismic-richtext` library: https://github.com/prismicio/prismic-richtext/blob/b4822be30aa691e9d66dbeb8d215ae957c95d06a/src/serialize.ts#L5

I also included a list of element types

Reply:
Makes sense.